### DISCUSSION TITLE
circular polarisation

### DISCUSSION THREAD

**Discussion by aodhanmci** [BLAST-WarpX/warpx#4424] [Q&A] [answered]
hi, is there a default method to control the phase of each polarisation in the laser antenna in order to make circular polarisation? Or should I define my own field in each polarisation direction and add my own phase? or is this all easier to do with Lasy?

thanks,
Aodhan

**Comment by aodhanmci** [comment, marked as answer]
never mind, simplest is just loading 2 laser profiles with orthogonal polarisations and changing the phase with phi0

**Reply by ax3l** [reply]
Perfect, yes that's how I usually do it as well.
